### PR TITLE
Turkish translations of many components.

### PR DESCRIPTION
**Pull request by ercanersoy**
[NTVDM] Fix Turkish translations.
[LOCALMON] Fix Turkish translations.
[USER32] Fix Turkish translations.
[CONSRV] Fix Turkish translations.
[CONSRW_NEW] Fix Turkish translations.
[USERSRV] Fix and add Turkish translations.
[BLACKSHADE.MSSTYLES] Fix Turkish translations.
[LAUTUS.MSSTYLES] Fix Turkish translations.
[MODERN.MSSTYLES] Fix Turkish translations.
[RUNDLL32] Fix Turkish translations.
[USERINIT] Fix and add Turkish translations.
[WINLOGON] Fix and add Turkish translations.

### CODE REVIEW

File: win32ss/user/winsrv/usersrv/lang/tr-TR.rc [hunk 1]
```diff
@@ -9,7 +9,7 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     LTEXT "Program sonlandırılıyor... Lütfen bekleyiniz.", IDC_STATIC, 7, 7, 186, 11
     CONTROL "İlerleme", IDC_PROGRESS, "msctls_progress32", WS_BORDER, 7, 20, 186, 13
-    LTEXT "Programı hemen kapatmayı seçerseniz, tüm saklanmamış veriyi yitireceksiniz. Programdan şimdi çıkmak için Şimdi Sonlandır'a tıklayınız.", IDC_STATIC, 7, 40, 186, 26
+    LTEXT "Programı hemen kapatmayı seçerseniz, tüm saklanmamış veriyi yitireceksiniz. Programdan şimdi çıkmak için \"Şimdi Sonlandır\"'a tıklayınız.", IDC_STATIC, 7, 40, 186, 26
```
Comment (peterooch) [inline win32ss/user/winsrv/usersrv/lang/tr-TR.rc]: Use `""` instead of `\"` , also in line 25.

File: base/system/winlogon/lang/tr-TR.rc [hunk 2]
```diff
@@ -18,37 +18,37 @@ CAPTION "Kullanıcı Arayüzü Yanlışlığı"
...
 IDD_SYSSHUTDOWN DIALOGEX 50, 50, 180, 140
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_VISIBLE | WS_CAPTION | WS_POPUP
-CAPTION "System Shutdown"
+CAPTION "Sistemi Kapatma"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON IDI_WARNING, IDC_SYSSHUTDOWNICON, 6, 7, 18, 20
-    LTEXT "A system shutdown has been initiated. Please save all your work and terminate your session. All unsaved work will be lost when the system shuts down.", -1, 38, 7, 135, 40
-    LTEXT "The system shuts down in:", -1, 38, 50, 90, 8
+    LTEXT "Bir sistem kapatma başlatıldı. Lütfen bütün işlerinizi kaydediniz ve oturumunuzu sonlandırınız.Bütün kaydedilmemiş işler, sistem kapatılırken kaybedilir.", -1, 38, 7, 135, 40
```
Comment (HBelusca) [inline base/system/winlogon/lang/tr-TR.rc]: Missing space after the full-stop in "sonlandırınız.Bütün"

File: win32ss/user/winsrv/usersrv/lang/tr-TR.rc [hunk 3]
```diff
@@ -22,22 +22,22 @@ BEGIN
...
 
 STRINGTABLE
 BEGIN
-    IDS_SYSTEM_PROCESS          "System Process"
-    IDS_UNKNOWN_SOFTWARE_EXCEPT "unknown software exception"
-    IDS_OK_TERMINATE_PROGRAM    "Click on OK to terminate the program."
-    IDS_CANCEL_DEBUG_PROGRAM    "Click on CANCEL to debug the program."
+    IDS_SYSTEM_PROCESS          "Sistem İşlemi"
+    IDS_UNKNOWN_SOFTWARE_EXCEPT "bilinmeyen yazılım istisnası"
+    IDS_OK_TERMINATE_PROGRAM    "Programı sonlandırmak için Tamam'a basınız."
```
Comment (HBelusca) [inline win32ss/user/winsrv/usersrv/lang/tr-TR.rc]: Does this string (and the next one) mentions the names of the buttons to be clicked for performing the actions?

**Comment by ercanersoy** [comment]
I fixed these errors.

File: win32ss/user/winsrv/usersrv/lang/tr-TR.rc [hunk 4]
```diff
@@ -36,8 +36,8 @@ END
 
 STRINGTABLE
 BEGIN
+    IDS_CANCEL_DEBUG_PROGRAM    "Programda hata ayıklamak için İptal'e basınız."
```
Comment (HBelusca) [inline win32ss/user/winsrv/usersrv/lang/tr-TR.rc]: why moving it above? please keep it at the end.

**Comment by ercanersoy** [comment]
I updated "tr-TR.rc". I have misunderstood this review.

**Pull request merged**